omicron_server/__main__.py

```diff
@@ -50,7 +61,23 @@ def create_root_user(session):
         session.add(user)
 
 if __name__ == "__main__":
```

Comment:
Even though this ```if``` block will protect us running the code in it, it comes with the problem that it becomes a lot more difficult to test this code. Can we put all the business logic in this if block into a ```main``` function and run that instead?

Instead of this

```python
import docopt

if __name__ == '__main__':
    arguments = docopt(__doc__)
    if arguments['HOST'] == 'DEFAULT':
        host = conf.IP_ADDRESS
```

we could do

```python
import docopt

def main():
    arguments = docopt(__doc__)
    if arguments['HOST'] == 'DEFAULT':
        host = conf.IP_ADDRESS

if __name__ == '__main__':
    main()
```